### ISSUE TITLE
How to access the ROI?

### ISSUE THREAD

**Issue by joicejoseph3198** [ultralytics/yolov5#4207]
## ❔Question

Is there a way to access the part of the image that is enclosed by the bounding box? Could you point to the part of the code that would allow me to do that? Or give suggestions to extract the portion of image surrounded by the bbox. Any help would be appreciated.

**Comment by glenn-jocher** [comment]
@joicejoseph3198 `python detect.py --save-crop`

**Comment by iceisfun** [comment]
Once you get the xyxy

X1 = int(xyxy[0])
Y1 = int(xyxy[1])
X2 = int(xyxy[2])
Y2 = int(xyxy[3])

roi = img[Y1:Y2, X1:X2, :].copy()

**Comment by joicejoseph3198** [comment]
> 
> 
> Once you get the xyxy
> 
> X1 = int(xyxy[0])
> Y1 = int(xyxy[1])
> X2 = int(xyxy[2])
> Y2 = int(xyxy[3])
> 
> roi = img[Y1:Y2, X1:X2, :].copy()

There is this bit of code in detect.py 
 # Process predictions
        for i, det in enumerate(pred):  # detections per image
            if webcam:  # batch_size >= 1
                p, s, im0, frame = path[i], f'{i}: ', im0s[i].copy(), dataset.count
            else:
                p, s, im0, frame = path, '', im0s.copy(), getattr(dataset, 'frame', 0)

            p = Path(p)  # to Path
            save_path = str(save_dir / p.name)  # img.jpg
            txt_path = str(save_dir / 'labels' / p.stem) + ('' if dataset.mode == 'image' else f'_{frame}')  # img.txt
            s += '%gx%g ' % img.shape[2:]  # print string
            gn = torch.tensor(im0.shape)[[1, 0, 1, 0]]  # normalization gain whwh
            imc = im0.copy() if save_crop else im0  # for save_crop

is imc the roi basically?
